connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -220,7 +224,8 @@ private Long ensureLeaderConfig(long maxOffset) {
                 connectorAssignments.put(connectorAssignedTo, memberConnectors);
             }
             memberConnectors.add(connectorId);
-
+        }
+        for (String connectorId : sorted(configSnapshot.connectors())) {
```

Comment:
nit: may as well store `sorted(configSnapshot.connectors())` in a variable since it's used twice.